vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManagerTest.java

```diff
@@ -146,24 +189,88 @@ public void test_updateContentOrder_success() throws BlockDoesNotExistException
     public void test_updateContentOrder_forNonExistentId() throws BlockDoesNotExistException {
         ContentBlock contentBlock1 = new ContentBlock();
         contentBlock1.setId("notARealId");
         contentBlock1.setContentOrder(Integer.valueOf(1));
-        
+
         List<ContentBlock> contentBlocks = new ArrayList<>();
         contentBlocks.add(contentBlock1);
-        
+
         when(contentBlockRepository.findById("notARealId")).thenReturn(Optional.empty());
         managerToTest.updateContentOrder(contentBlocks);
     }
 
+    @Test
+    public void test_deleteVideoBlockById_success() throws BlockDoesNotExistException {
+        String videoBlockId = "videoBlockId_2";
+        Optional<ContentBlock> contentBlockOptional = Optional.of(contentBlock);
+        when(contentBlockRepository.findById(videoBlockId)).thenReturn(contentBlockOptional);
+        when(contentBlockRepository.findBySlide_IdAndContentOrderGreaterThan("slideId_1", Integer.valueOf(1)))
+                .thenReturn(contentBlockList);
+        managerToTest.deleteVideoBlockById(videoBlockId, "slideId_1");
+        Mockito.verify(videoBlockRepo).deleteById(videoBlockId);
+        List<Integer> contentOrderList = contentBlockList.stream().map(contentBlock -> contentBlock.getContentOrder())
+                .collect(Collectors.toList());
+
+        assertEquals(Integer.valueOf(1), contentOrderList.get(0));
+        assertEquals(Integer.valueOf(2), contentOrderList.get(1));
+
+        Mockito.verify(contentBlockRepository).saveAll(contentBlockList);
+    }
+
+    @Test(expected = BlockDoesNotExistException.class)
+    public void test_deleteVideoBlockById_forNonExistentId() throws BlockDoesNotExistException {
+        String videoBlockId = "notARealId";
+        Optional<ContentBlock> contentBlockOptional = Optional.of(contentBlock);
+        when(contentBlockRepository.findById("notARealId")).thenReturn(contentBlockOptional);
+        Mockito.doThrow(EmptyResultDataAccessException.class).when(videoBlockRepo).deleteById(videoBlockId);
+        managerToTest.deleteVideoBlockById(videoBlockId, "slideId_1");
+    }
+
+    @Test
+    public void test_deleteVideoBlockById_whenIdIsNull() throws BlockDoesNotExistException {
+        String videoBlockId = null;
+        managerToTest.deleteTextBlockById(null, "slideId_1");
+        Mockito.verify(videoBlockRepo, Mockito.never()).deleteById(videoBlockId);
+    }
+
+    @Test
+    public void test_getVideoBlock_success() throws BlockDoesNotExistException {
+        String videoBlockId = "videoBlockId_3";
+        Optional<VideoBlock> contentBlockOptional = Optional.of(videoBlock);
+        when(videoBlockRepo.findById(videoBlockId)).thenReturn(contentBlockOptional);
+        IVideoBlock videoBlock = managerToTest.getVideoBlock(videoBlockId);
+        assertEquals(videoBlockId, videoBlock.getId());
+    }
+    
+    @Test
+    public void test_createVideoBlock_withURLsuccess()
```

Comment:
what about other scenarios with an uploaded videos, or when this method fails? Needs more test cases.